src/js/utils/constants.js

```diff
@@ -9,6 +9,12 @@ export const EQU = "=";
 export const AC = "AC";
 export const OPERATORS = [ADD, SUB, MUL, DIV, EQU, AC];
 
+export const CALC = ".calculator";
+export const TOTAL = "#total";
+export const DIGITS = ".digits";
+export const OPERATIONS = ".operations";
+export const MODIFIER = ".modifiers";
```

Comment:
```suggestion
export const Selectors = {
    CALC: ".calculator",
    TOTAL: "#total",
    DIGITS: ".digits",
    OPERATIONS: ".operations",
    MODIFIER: ".modifiers",
}
```
이런식으로 selector라는 namespace로 묶어주면 어떨까요?

Reply:
앗 그렇네요! 연관성 있는 변수들끼리 묶어주니 더 깔끔해 보이는 것 같아요 👍